test/test_prototype_transforms_consistency.py

```diff
@@ -840,3 +840,681 @@ def test_aa(self, inpt, interpolation):
         output = t(inpt)
 
         assert_equal(expected_output, output)
+
+
+class TestRefDetTransforms:
+    def make_datapoints(self, with_mask=True):
+        size = (600, 800)
+
+        pil_image = PIL.Image.new("RGB", size[::-1], 123)
+        target = {
+            "boxes": make_bounding_box(image_size=size, format="XYXY", extra_dims=(22,), dtype=torch.float),
+            "labels": features.Label(torch.randint(0, 81, size=(22,))),
```

Comment:
Is the number of categories important here? If not I would prefer to use the common utilities to create the label

```suggestion
            "labels": make_label(size=(22,)),
```

Note that #5879 introduces functionality to specify `categories=80`.

Reply:
After #5879 the call should look like

```suggestion
            "labels": make_label(extra_dims=(22,), categories=80),
```

Reply:
Finally, I think it is unimportant to specify 80 categories instead of using default values.